UPGRADE-2.0.md

```diff
@@ -85,13 +127,17 @@
   After:
 
   ```php
-  $client->getConfig()->getRelease();
-  $client->getConfig()->setRelease(...);
+  use Sentry\Hub;
+
+  $options = Hub::getCurrent()->getClient()->getOptions();
```

Comment:
Actually I wrote it like that because it resembles what was previously being done, that is changing the value of the option on-the-fly instead of at the client initialization